.github/workflows/ci.yml

```diff
@@ -20,8 +20,11 @@ jobs:
           brew install shfmt shellcheck
           shfmt -d .
 
+      - name: spellcheck
```

Comment:
```suggestion
      - name: shellcheck
```

Reply:
My brain is just really hung up on the word spellcheck this morning...